keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -874,46 +869,29 @@ Think about adding additional work or introducing new steps in between
 
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
-Mutating API calls that opt-in to validation will be slower ([initial
-benchmarks](https://github.com/kubernetes/kubernetes/pull/104433#issuecomment-901398507)
-estimate ~20% slower 25-30% more memory consumption)
+Mutating API calls that opt-in to validation will be slower.
```

Comment:
clarify that this performance hit is limited to json/yaml requests, and that the majority of high-frequency requests made by system components use protobuf

Reply:
done